Directory.Build.targets

```diff
@@ -84,8 +84,11 @@
     <FrameworkReference Include="Microsoft.Windows.SDK.NET.Ref" IsImplicitlyDefined="true" Pack="false" PrivateAssets="All" />
   </ItemGroup>
   <PropertyGroup>
-    <WindowsSdkPackageVersion>10.0.19041.34</WindowsSdkPackageVersion>   
+    <WindowsSdkPackageVersion>10.0.19041.34</WindowsSdkPackageVersion>
```

Comment:
Looks like this and above got uncommented during merge.  This is already set to 44 in the props.

Reply:
Oops -- lemme remove this